backend/requests.rest

```diff
@@ -26,12 +26,22 @@ GET http://localhost:3000/posts
 ### Delete all posts
 DELETE http://localhost:3000/posts
 
-### Add a post
-POST http://localhost:3000/posts/add
+### Add a new post
+POST http://localhost:3000/posts
```

Comment:
the path is still with /add and also the body is still the same with "timestamp" and "text"